src/main/resources/application.yml

```diff
@@ -11,3 +11,6 @@ spring:
     password: postgres
   flyway:
     baseline-on-migrate: true
+
+app_host: localhost:8080
+app_env: dev
```

Comment:
this is used normaly in a conf name application-dev.yml and then as ENV variable